core/src/main/java/org/kiwix/kiwixmobile/core/entity/LibraryNetworkEntity.kt

```diff
@@ -0,0 +1,102 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2019 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.core.entity
+
+import org.simpleframework.xml.Attribute
+import org.simpleframework.xml.ElementList
+import org.simpleframework.xml.Root
+import java.io.File
+import java.io.Serializable
+import java.util.LinkedList
+
+@Root(name = "library")
+class LibraryNetworkEntity {
+  @field:ElementList(name = "book", inline = true, required = false)
+  var book: LinkedList<Book>? = null
+
+  @field:Attribute(name = "version", required = false)
+  var version: String? = null
+
+  @Root(name = "book", strict = false)
+  class Book : Serializable {
+    @kotlin.jvm.JvmField @field:Attribute(name = "id", required = false)
+    var id: String = ""
+
+    @kotlin.jvm.JvmField @field:Attribute(name = "title", required = false)
```

Comment:
[nit] We can remove all the package names and leave the only annotation for the sake of code readablity.

Reply:
hi @gouri-panda ,
done